timely_beliefs/tests/test_db_subclassing.py

```diff
@@ -136,3 +136,6 @@ def test_custom_source_and_beliefs_with_mixin(db):
 
     assert "my_belief_source" in db.tables.keys()
     assert "my_timed_belief" in db.tables.keys()
+
+    bdf = JoyfulBeliefInCustomTable.query(session, sensor)
```

Comment:
Is this supposed to be `query_all`? If so, why does this assertion work? Is `bdf` maybe an `sqlachemy.Query` object instead of a `BeliefsDataFrame`?

Reply:
I used `query` on purpose, to have a test that triggers the corresponding FutureWarning. I've been following this strategy for all deprecations, for two reasons:

1. The test suite will show FutureWarnings for upcoming deprecations, which keeps reminding developers to actually deprecate at some point.
2. The developer that implements the actual deprecation will break the test, and is therefore made aware of the fact that dependent code may break (and hopefully, reflect on whether enough time has passed since the deprecation was announced).

I'm glad you noticed this, because giving it some more thought I can come up with a few things to improve this strategy:

- Add a todo to the assert statement to amend the upcoming deprecation.
- Note the version at which the deprecation was announced.